### PR TITLE
Rm auth email domain

### PR DESCRIPTION
**Pull request by emileswarts**

### CODE REVIEW

File: govwifi-api/user-signup-api-cluster.tf [hunk 1]
```diff
@@ -60,6 +60,7 @@ EOF
 }
 
 data "aws_s3_bucket" "admin-bucket" {
+  count  = "${var.user-signup-enabled}"
```
Comment (kjdchapman) [inline govwifi-api/user-signup-api-cluster.tf]: What does this do?

Comment (cybojenix) [inline govwifi-api/user-signup-api-cluster.tf]: is this definitely the only thing it's being used for?

Comment (cybojenix) [reply govwifi-api/user-signup-api-cluster.tf]: It prevents the data resource being created when user signup isn't enabled.
Assume this resolves the issue of querying the resource with a bucket name set to a blank string

Comment (kjdchapman) [reply govwifi-api/user-signup-api-cluster.tf]: `count` / `enabled` caused the type checker in my brain to explode.

I mostly get it, just wanted to make sure this was on purpose etc.

Comment (cybojenix) [reply govwifi-api/user-signup-api-cluster.tf]: Yeah, it's not always obvious.
As a flag, I prefer '-enabled' as it should amount to 1 or 0 (true/false). 
Using '-count' implies there can potentially be more than one resource.

**Pull request merged**